pkg/authentication/firebase.go

```diff
@@ -0,0 +1,46 @@
+package authentication
+
+import (
+	"context"
+	"firebase.google.com/go/v4/auth"
+)
+
+// FirebaseTokenVerifier contains a method that allows verifying a Token signed by Firebase.
+type FirebaseTokenVerifier interface {
+	VerifyIDToken(ctx context.Context, idToken string) (*auth.Token, error)
```

Comment:
Isn't it a little bit repetitive to include the same information in both the method and the interface?

Reply:
In this case it is a bit repetitive, but only because the interface only contains a single method. Philosophically though, the method documentation can contain implementation detail recommendations (e.g. input values, standard return values), etc.

Reply:
I included information about `idToken`. Care to review it?

Reply:
Excellent. Thank you!